conmon/cmsg.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2016 SUSE LLC
```

Comment:
@jawnsy In general, it's just the style within SUSE (and the GNU project, and Linux, and systemd, and ...) to include a copyright statement in all files (and also RedHat AFAIK, though in the Go community people appear to not like including copyright statements in comments because comments actually affect compilation). That's why I included it in the original file, so sorry for any inconvenience.